styles.css

```diff
@@ -0,0 +1,94 @@
+/* stylelint-disable no-missing-end-of-source-newline */
+html {
+  height: 4910px;
+  width: 320px;
+  left: 6158px;
+  top: -2164px;
```

Comment:
- [ ] Kindly check the HTML container's margin, padding, and width that affect the mobile screen sizes greater than 320 px width.

***Figma design***
![Screenshot from 2022-12-01 21-03-53](https://user-images.githubusercontent.com/65129228/205134019-4f12b0a9-6908-4184-8b2d-5c2929c6881d.png)


***Your design***
![Screenshot from 2022-12-01 21-37-42](https://user-images.githubusercontent.com/65129228/205134100-eed4e708-2cbb-4ff7-bfff-bcd005230c21.png)